romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -328,7 +328,7 @@ properties:
             title: Name of the filter element used
             type: string
             enum: [F062, F087, F106, F129, W146, F158, F184, GRISM, PRISM, DARK, ENGINEERING]
-        required: [detector, optical_element, name]
```

Comment:
This field is still required in the flat schema, right?  Should we make it required there and in other schemas that need it?

Reply:
I have sent a mail to TPTB about how best to implement that for subsets of models that require them.

Reply:
TPTB = The Powers That Be aka @ddavis-stsci, @nden, and @sosey

Reply:
@sosey, Did you get a chance to see the mail I sent you three last week where I suggested making "requirement profile" schema files that could then be included by appropriate schema files? Doing so could eliminate having to duplicate requirements across many schema files.

Reply:
Taken to RCAL-126